### PR TITLE
FEATURE: Add with google logic

### PR DESCRIPTION
**Pull request by DavMovsesyan2002**

### CODE REVIEW

File: src/pages/index.tsx [hunk 1]
```diff
@@ -31,6 +32,13 @@ const data1 = [
 
 const Home = () => {
   const [t] = useTranslation()
+  const router = useRouter()
+
+  useEffect(() => {
+    if (router.query) {
+      localStorage.setItem('accessToken', router.query.accessToken as string)
+    }
+  }, [router])
```
Comment (ArthurO2002) [inline src/pages/index.tsx:41]: Let's redirect to profile page @Zt0 @DavMovsesyan2002

File: src/manager/authentication/authManager.ts [hunk 2]
```diff
@@ -42,10 +42,7 @@ const authManager = {
     )
   },
   googleSignIn() {
-    // TODO: use env variables for this call
-    return axiosInstance.get<null, IAxiosResponse<null>>(
-      `http://localhost:2002/users/v1/auth/google`
-    )
+    return window.open(`http://localhost:2002/users/v1/auth/google`)
   },
```
Comment (ArthurO2002) [inline src/manager/authentication/authManager.ts:46]: Get this from Env

**Review by ArthurO2002 — DISMISSED** [review]

File: src/pages/index.tsx (hunk 1 above)
Comment (DavMovsesyan2002) [reply src/pages/index.tsx]: Done

File: src/manager/authentication/authManager.ts (hunk 2 above)
Comment (DavMovsesyan2002) [reply src/manager/authentication/authManager.ts]: Done

**Review by ArthurO2002 — DISMISSED** [review]

**Review by ArthurO2002 — APPROVED** [review]

**Pull request merged**